conf/boards/lisa_mxs_1.0.makefile

```diff
@@ -0,0 +1,28 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# lisa_mxs_1.0.makefile
+#
+# https://wiki.paparazziuav.org/wiki/Lisa/MXS_v1.0
+#
+
+BOARD=lisa_mxs
+BOARD_VERSION=1.0
+BOARD_CFG=\"boards/$(BOARD)_$(BOARD_VERSION).h\"
+
+ARCH=stm32
```

Comment:
You're adding a lot of file related to ChibiOS, but still selecting the stm32 arch. And I don't think it is currently working without ChibiOS at the moment as some definition are missing. I think you should limit your pull request to adding the basic support of the lisa_mxs board and its flashing option, and add ChibiOS support later it needed when it is ready.

Reply:
Removed those files. I just copied it assuming it already worked.